bazel/foreign_cc/vpp_vcl.patch

```diff
@@ -70,15 +70,15 @@ index 8415c28fb..a017d9fc6 100755
 +
  import ply.lex as lex
  import ply.yacc as yacc
-
+ 
 diff --git src/vcl/CMakeLists.txt src/vcl/CMakeLists.txt
-index e6d8f98ff..600ef4d5f 100644
+index 610b422d1..c5e6f8ca8 100644
 --- src/vcl/CMakeLists.txt
 +++ src/vcl/CMakeLists.txt
-@@ -30,6 +30,8 @@ add_vpp_library(vppcom
-   api_headers
- )
-
+@@ -35,6 +35,8 @@ if (LDP_HAS_GNU_SOURCE)
+   add_compile_definitions(HAVE_GNU_SOURCE)
+ endif(LDP_HAS_GNU_SOURCE)
```

Comment:
This is already part of the file so is it needed?

Reply:
This should be diff re-location line number, the final diff in *.patch is:

--- src/vcl/CMakeLists.txt
+++ src/vcl/CMakeLists.txt
@@ -35,6 +35,8 @@ if (LDP_HAS_GNU_SOURCE)
   add_compile_definitions(HAVE_GNU_SOURCE)
 endif(LDP_HAS_GNU_SOURCE)
 
+file(COPY vppcom.h DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+

Reply:
Ow, makes sense!